src/main/java/wooteco/subway/ui/GlobalExceptionHandler.java

```diff
@@ -16,15 +20,32 @@
 @ControllerAdvice
 public class GlobalExceptionHandler {
 
+    @ExceptionHandler(BindException.class)
+    private ResponseEntity<ErrorResponse> handleValidException(BindException e) {
+        String message = e.getBindingResult()
+                .getAllErrors()
+                .stream()
+                .map(DefaultMessageSourceResolvable::getDefaultMessage)
+                .collect(Collectors.joining(","));
+
+        return ResponseEntity.badRequest().body(new ErrorResponse(message));
+    }
+
     @ExceptionHandler({DuplicateException.class, BlankArgumentException.class,
-        UnaddableSectionException.class, UndeletableSectionException.class, UnreachablePathException.class,
-        UnsplittableException.class, UnmergeableException.class})
+            UnaddableSectionException.class, UndeletableSectionException.class, UnreachablePathException.class,
+            UnsplittableException.class, UnmergeableException.class, RidiculousAgeException.class})
     private ResponseEntity<ErrorResponse> handleExceptionToBadRequest(Exception e) {
+        System.out.println("########## " + e.getMessage());
         return ResponseEntity.badRequest().body(new ErrorResponse(e.getMessage()));
     }
 
     @ExceptionHandler(NotFoundException.class)
-    private ResponseEntity<Void> handleExceptionToNotFound() {
+    private ResponseEntity<Void> handleExceptionToNotFound(Exception e) {
```

Comment:
```suggestion
    private ResponseEntity<Void> handleExceptionToNotFound(NotFoundException e) {
```
`NotFoundException`으로 받는 게 명확하지 않을까요?

Reply:
네 더 정학하게 나타내 주기 위해 NotFoundException로 변경하였습니다!